app/models/settings.js

```
@@ -20,6 +20,7 @@ export function init () {
     timeout: 0,
     validateSSL: true,
     forceVerticalLayout: false,
+    autoHideMenuBar: true,
```

Note that this default will hide everyone's menu bars after the next update. Is the alt-to-show-menu-bar behavior common enough that users won't be confused? (I'm a Mac user if you can't tell 😛)